src/frontend/src/test-e2e/util.ts

```diff
@@ -176,6 +181,98 @@ export async function addCustomCommands(
       }
     )
   );
+
+  await browser.addCommand(
```

Comment:
```suggestion
  // This retrieves previously created credentials, see https://www.w3.org/TR/webauthn-2/#sctn-automation-get-credentials
  await browser.addCommand(
```

I wouldn't think of looking in the object to find the link, wdyt?

Reply:
Out of curiosity, what are all those "variables" and "parameters" and "refs" etc? Are they actually read? Or just used in error messages?

Reply:
Yes, parameters and variables are read by the selenium driver. What it essentially does is create a HTTP request out of this information. `description` and `ref` is for humans only, but I figured the additional information might be useful (and also following the existing example).